main.tscn

```diff
@@ -47,7 +47,6 @@ polygon = PackedVector2Array(-31, 0, 0, 16, 32, 0, 0, -16)
 [node name="Plot2" type="StaticBody2D" parent="TileMap" groups=["plots"]]
 position = Vector2(64, 128)
 script = ExtResource("4_wgdl0")
-built = true
```

Comment:
You remove this for a reason? Or is it a weird holdover?

This is the plots already being built.

Reply:
Uh
no not deliberately, I might have fucked something in godot